src/main/java/org/apache/commons/imaging/formats/icns/IcnsDecoder.java

```diff
@@ -21,6 +21,7 @@
 import java.util.List;
 
 import org.apache.commons.imaging.ImageReadException;
+import org.apache.commons.imaging.Imaging;
```

Comment:
This could lead to a circular dependency between the package `oac.imaging`, and the type `Imaging`, with this file. So I will try to use the parser directly instead 👍